dockerfiles/agents/Dockerfile.agent

```diff
@@ -7,7 +7,7 @@ ARG PROTOBUF_VERSION=5.28.1
 RUN pip install --no-cache-dir protobuf==${PROTOBUF_VERSION}
 
 # install nats-events plugin
-RUN pip install --no-cache-dir git+https://github.com/didx-xyz/aries-acapy-plugins@test-refactor-multitenant-manager-errors#subdirectory=nats_events
+RUN pip install --no-cache-dir git+https://github.com/didx-xyz/aries-acapy-plugins@nats-reconnect#subdirectory=nats_events
```

Comment:
I just want to update this with a tagged release of the plugin, and then we can merge 👍 will do that later today